intercom/assets/logs/intercom.yaml

```diff
@@ -0,0 +1,291 @@
+    source: log
+  - groups:
+      - Geoip
+    name: Country Name
+    path: network.client.geoip.country.name
+    source: log
+  - groups:
+      - Geoip
+    name: Subdivision ISO Code
+    path: network.client.geoip.subdivision.iso_code
+    source: log
+  - groups:
+      - Geoip
+    name: Subdivision Name
+    path: network.client.geoip.subdivision.name
+    source: log
+pipeline:
+  type: pipeline
+  name: Intercom
+  enabled: true
+  filter:
+    query: "source:intercom"
+  processors:
+    - name: Lookup for `data.item.type` to `service`
+      enabled: true
+      source: data.item.type
+      target: service
+      lookupTable: |-
+        admin_activity_log_event, activity
+        event, data_event
+        conversation, conversation
+        content_stat.news_item, news_item
+        ticket, ticket
+        ticket_ticket_part, ticket
+      type: lookup-processor
+    - type: service-remapper
+      name: Define `service` as the official service of the log
+      enabled: true
+      sources:
+        - service
+    - type: pipeline
+      name: "Conversation"
+      enabled: true
+      filter:
+        query: "service:conversation"
+      processors:
+        - type: arithmetic-processor
+          name: Convert `data.item.updated_at` from second to millisecond EPOCH UNIX
+            format and map to `updated_at_ms`
+          enabled: true
+          expression: data.item.updated_at*1000
+          target: updated_at_ms
+          replaceMissing: false
+        - type: date-remapper
+          name: Define `updated_at_ms` as the official date of the log
+          enabled: true
+          sources:
+            - updated_at_ms
+        - type: attribute-remapper
+          name: Map `data.item.source.author.id` to `usr.id`
+          enabled: true
+          sources:
+            - data.item.source.author.id
+          sourceType: attribute
+          target: usr.id
+          targetType: attribute
+          preserveSource: true
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: Map `data.item.source.author.email` to `usr.email`
+          enabled: true
+          sources:
+            - data.item.source.author.email
+          sourceType: attribute
+          target: usr.email
+          targetType: attribute
+          preserveSource: true
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: Map `data.item.source.author.name` to `usr.name`
+          enabled: true
+          sources:
+            - data.item.source.author.name
+          sourceType: attribute
+          target: usr.name
+          targetType: attribute
+          preserveSource: true
+          overrideOnConflict: false
+        - type: attribute-remapper
+          name: Map `data.item.custom_attributes.Copilot used` to
+            `data.item.custom_attributes.copilot_used`
+          enabled: true
+          sources:
+            - data.item.custom_attributes.Copilot used
```

Comment:
Is this source correct?  In the test sample, I see this as `data.item.custom_attributes.copilot_used`.  If the source and the target are the same, does this need to be mapped at all?

Reply:
I have modified the sample response to address the test case.